pom.xml

```diff
@@ -1,32 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-The contents of this file are subject to the terms of the Common Development and
-Distribution License (the License). You may not use this file except in compliance with the
-License.
+    The contents of this file are subject to the terms of the Common Development and
+    Distribution License (the License). You may not use this file except in compliance with the
+    License.
 
-You can obtain a copy of the License at legal/CDDLv1.0.txt. See the License for the
-specific language governing permission and limitations under the License.
+    You can obtain a copy of the License at legal/CDDLv1.0.txt. See the License for the
+    specific language governing permission and limitations under the License.
 
-When distributing Covered Software, include this CDDL Header Notice in each file and include
-the License file at legal/CDDLv1.0.txt. If applicable, add the following below the CDDL
-Header, with the fields enclosed by brackets [] replaced by your own identifying
-information: "Portions Copyrighted [year] [name of copyright owner]".
+    When distributing Covered Software, include this CDDL Header Notice in each file and include
+    the License file at legal/CDDLv1.0.txt. If applicable, add the following below the CDDL
+    Header, with the fields enclosed by brackets [] replaced by your own identifying
+    information: "Portions Copyrighted [year] [name of copyright owner]".
 
-Copyright 2014-2015 ForgeRock AS.
-Portions Copyright 2017 Wren Security.
+    Copyright 2014-2015 ForgeRock AS.
+    Portions Copyright 2017-2021 Wren Security.
```

Comment:
Is outdated.

Reply:
This is correct as the PR is from the last year.